### PR TITLE
Adds registry.access.redhat.com to firewall allowlist list

### PR DESCRIPTION
**Pull request by stevsmit**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OCPBUGS-37868

Link to docs preview:
https://79926--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/install_config/configuring-firewall.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/configuring-firewall.adoc [hunk 1]
```diff
@@ -40,7 +40,11 @@ If your environment has a dedicated load balancer in front of your {product-titl
 
 |`access.redhat.com` ^[1]^
 |443
-|Hosts all the container images that are stored on the Red Hat Ecosytem Catalog, including core container images.
+|Hosts all the container images that are stored on the Red Hat Ecosystem Catalog, including core container images.
```
Comment (patrickdillon) [inline modules/configuring-firewall.adoc:43]: We should change this description for `access.redhat.com` to distinguish it from `registry.access.redhat.com`. 
It seems `access.redhat.com` holds the signature store while `registry.access.redhat.com` holds the registry. 
So I would change this description to say:

```suggestion
|Hosts a signature store that a container client requires for verifying images pulled from `registry.access.redhat.com`.
```

File: modules/configuring-firewall.adoc [hunk 2]
```diff
@@ -80,10 +84,12 @@ If your environment has a dedicated load balancer in front of your {product-titl
 |===
 +
 --
-1. In a firewall environment, ensure that the `access.redhat.com` resource is on the allowlist. This resource hosts a signature store that a container client requires for verifying images when pulling them from `registry.access.redhat.com`.
+1. In a firewall environment, ensure that the `access.redhat.com` and `registry.access.redhat.com` resources are on the allowlist. These resources host a signature store that a container client requires for verifying images when pulling them from `registry.access.redhat.com`.
```
Comment (patrickdillon) [inline modules/configuring-firewall.adoc:87]: Based on my other comment I think we can move this into the description of `access.redhat.com` and remove this node.

File: modules/configuring-firewall.adoc (hunk 1 above)
Comment (stevsmit) [reply modules/configuring-firewall.adoc:43]: Fixed. Thanks.

File: modules/configuring-firewall.adoc [hunk 3]
```diff
@@ -38,9 +38,13 @@ If your environment has a dedicated load balancer in front of your {product-titl
 |443
 |Provides core container images
 
-|`access.redhat.com` ^[1]^
+|`access.redhat.com`
 |443
-|Hosts all the container images that are stored on the Red Hat Ecosytem Catalog, including core container images.
+|Hosts all the container images that are stored on the Red Hat Ecosystem Catalog, including core container images. In a firewall environment, ensure this resource is on the allowlist, because it hosts a signature store that a container client requires for verifying images when pulling them from `registry.access.redhat.com`.
```
Comment (patrickdillon) [inline modules/configuring-firewall.adoc:43]: I don't think there's a preview available yet to verify but these look switched, i.e. this (`Hosts all the container images`) is the description for registry.access.redhat.com

**Comment by patrickdillon** [comment]
LGTM

**Pull request merged**